velox/functions/sparksql/tests/DecimalArithmeticTest.cpp

```diff
@@ -49,6 +49,16 @@ class DecimalArithmeticTest : public SparkFunctionBaseTest {
     assertEqualVectors(expected, evaluate(expr, makeRowVector(inputs)));
   }
 
+  template <TypeKind KIND>
+  void testDecimalUnaryminusFunction(
```

Comment:
It looks like 'testArithmeticFunction' could be reused, how do you think?

Reply:
Updated, thanks